advanced_functionality/huggingface_byo_scripts_and_data/huggingface-custom-text-summarizer.ipynb

```diff
@@ -0,0 +1,2886 @@
+{
```

Comment:
<p>Grammar/branding nit - be consistent with <code>Hugging Face</code> versus <code>HuggingFace</code> in markdown. I believe with a space is the correct name.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>